internal/operators/lvm/lvm_operator.go

```diff
@@ -83,7 +82,7 @@ func (o *operator) ValidateCluster(_ context.Context, cluster *common.Cluster) (
 		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetClusterValidationID(), Reasons: []string{message}}, nil
 	}
 
-	if swag.StringValue(cluster.HighAvailabilityMode) == models.ClusterHighAvailabilityModeFull {
+	if cluster.ControlPlaneCount == int64(3) {
```

Comment:
Lets change this condition to make more sense for clusters with more than 3 control plane nodes